# std header
std_msgs/Header header

# group name
string group_name

# workspace samples
WorkspacePoint[] points

# Store information about how the workspace has been sampled here
moveit_msgs/WorkspaceParameters parameters
float64 position_resolution
geometry_msgs/Quaternion[] orientations

